tests/nfa/nfa.cc

```diff
@@ -1021,17 +1020,17 @@ TEST_CASE("Mata::Nfa::serialize() and operator<<()")
 		aut.delta.add('s', 'd', 'q');
 		aut.delta.add('q', 'a', 'q');
 
-		Mata::Nfa::StateToStringMap state_dict =
+		Mata::Nfa::std::unordered_map<State, std::string> state_dict =
```

Comment:
We should probably just delete these test cases.

Reply:
The commented out tests were either deleted or uncommented and updated to pass.